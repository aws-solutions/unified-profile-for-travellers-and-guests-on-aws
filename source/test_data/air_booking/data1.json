{    "objectVersion": 6,    "modelVersion": "1.0",    "id": "FZHBNE",    "externalIds": null,    "lastUpdatedOn": "2021-09-05T08:05:44.658770036Z",    "createdOn": "2021-06-05T08:05:44.658770036Z",    "lastUpdatedBy": "Elroy Corwin",    "createdBy": "Derrick Walker",    "creationChannelId": "gds",    "price": {        "status": null,        "grandTotal": 0.0    },    "itinerary": {        "from": "DFW",        "to": "JFK",        "departureDate": "2021-08-20",        "departureTime": "09:59",        "arrivalDate": "2021-08-20",        "arrivalTime": "15:51",        "duration": "5h52m0s",        "segments": [            {                "rank": 0,                "from": "DFW",                "to": "ORD",                "departureDate": "2021-08-20",                "departureTime": "09:59",                "arrivalDate": "2021-08-20",                "arrivalTime": "11:08",                "flightNumber": "AA6357",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            },            {                "rank": 1,                "from": "ORD",                "to": "JFK",                "departureDate": "2021-08-20",                "departureTime": "13:43",                "arrivalDate": "2021-08-20",                "arrivalTime": "15:51",                "flightNumber": "AA3771",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            }        ],        "inventory": {            "totalSeats": 0,            "inventoryByFareClass": null        },        "status": "canceled"    },    "return": {        "from": "JFK",        "to": "DFW",        "departureDate": "2021-06-25",        "departureTime": "04:58",        "arrivalDate": "2021-06-25",        "arrivalTime": "14:59",        "duration": "10h1m0s",        "segments": [            {                "rank": 0,                "from": "JFK",                "to": "ORD",                "departureDate": "2021-06-25",                "departureTime": "04:58",                "arrivalDate": "2021-06-25",                "arrivalTime": "09:17",                "flightNumber": "AA4181",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            },            {                "rank": 1,                "from": "ORD",                "to": "DFW",                "departureDate": "2021-06-25",                "departureTime": "07:42",                "arrivalDate": "2021-06-25",                "arrivalTime": "14:59",                "flightNumber": "AA7253",                "inventory": {                    "totalSeats": 0,                    "inventoryByFareClass": null                },                "status": ""            }        ],        "inventory": {            "totalSeats": 0,            "inventoryByFareClass": null        },        "status": ""    },    "passengerInfo": {        "passengers": [            {                "modelVersion": "1.0",                "id": "2242121775",                "lastUpdatedOn": "2021-08-18T19:15:28.199814783Z",                "createdOn": "2021-04-18T19:15:28.199814783Z",                "lastUpdatedBy": "Keagan Mertz",                "createdBy": "Hobart Strosin",                "emails": [                    {                        "type": "",                        "address": "lysannegleason@thiel.info",                        "primary": true                    },                    {                        "type": "personal",                        "address": "luzcummings@hauck.net",                        "primary": true                    }                ],                "phones": [],                "addresses": [                    {                        "type": "shipping",                        "line1": "39874 New Junction port",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "Minneapolis",                        "state": {                            "code": "SD",                            "name": "Oregon"                        },                        "province": {                            "code": "",                            "name": ""                        },                        "postalCode": "76250",                        "country": {                            "code": "MF",                            "name": ""                        },                        "primary": true                    },                    {                        "type": "",                        "line1": "25581 Fields port",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "Milwaukee",                        "state": {                            "code": "OK",                            "name": "Vermont"                        },                        "province": {                            "code": "",                            "name": ""                        },                        "postalCode": "78312",                        "country": {                            "code": "BQ",                            "name": ""                        },                        "primary": true                    }                ],                "honorific": "Mx",                "firstName": "Westley",                "middleName": "Carter",                "lastName": "Brown",                "gender": "other",                "pronoun": "he",                "dateOfBirth": "2019-09-24",                "language": {                    "code": "ur",                    "name": ""                },                "nationality": {                    "code": "LS",                    "name": ""                },                "jobTitle": "Consultant",                "parentCompany": "Accenture",                "loyaltyPrograms": [                    {                        "id": "3454887990",                        "programName": "reward",                        "miles": 873251.0,                        "milesToNextLevel": 4637.0,                        "level": "bronze",                        "joined": "2021-04-18T19:15:28.199814783Z"                    },                    {                        "id": "4956081959",                        "programName": "elite",                        "miles": 195342.0,                        "milesToNextLevel": 1567.0,                        "level": "gold",                        "joined": "2021-04-18T19:15:28.199814783Z"                    }                ],                "identityProofs": null            },            {                "modelVersion": "1.0",                "id": "1107808714",                "lastUpdatedOn": "2022-03-08T17:24:34.476427356Z",                "createdOn": "2022-02-08T17:24:34.476427356Z",                "lastUpdatedBy": "Ferne VonRueden",                "createdBy": "Esta Hoppe",                "emails": [                    {                        "type": "business",                        "address": "lesliecronin@halvorson.org",                        "primary": false                    },                    {                        "type": "personal",                        "address": "dejuanbecker@jast.org",                        "primary": false                    },                    {                        "type": "business",                        "address": "donalddibbert@keebler.net",                        "primary": false                    }                ],                "phones": [                    {                        "type": "business",                        "number": "5808895143",                        "primary": false,                        "countryCode": 46                    }                ],                "addresses": [                    {                        "type": "",                        "line1": "673 Divide view",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "Charlotte",                        "state": {                            "code": "OR",                            "name": "Indiana"                        },                        "province": {                            "code": "",                            "name": ""                        },                        "postalCode": "21216",                        "country": {                            "code": "AX",                            "name": ""                        },                        "primary": false                    },                    {                        "type": "mailing",                        "line1": "6695 Forest berg",                        "line2": "more content line 2",                        "line3": "more content line 3",                        "line4": "more content line 4",                        "city": "Colorado Springs",                        "state": {                            "code": "WI",                            "name": "Indiana"                        },                        "province": {                            "code": "",                            "name": ""                        },                        "postalCode": "99917",                        "country": {                            "code": "VC",                            "name": ""                        },                        "primary": false                    }                ],                "honorific": "Mrs",                "firstName": "Yadira",                "middleName": "Melody",                "lastName": "Hayes",                "gender": "female",                "pronoun": "",                "dateOfBirth": "1912-09-26",                "language": {                    "code": "ss",                    "name": ""                },                "nationality": {                    "code": "TD",                    "name": ""                },                "jobTitle": "Planner",                "parentCompany": "Way Better Patents",                "loyaltyPrograms": [],                "identityProofs": null            }        ]    },    "paymentInformation": {        "paymentType": "bank_account",        "ccInfo": {            "token": "",            "cardType": "",            "cardExp": "",            "cardCvv": "",            "expiration": "",            "name": "",            "address": {                "type": "",                "line1": "",                "line2": "",                "line3": "",                "line4": "",                "city": "",                "state": {                    "code": "",                    "name": ""                },                "province": {                    "code": "",                    "name": ""                },                "postalCode": "",                "country": {                    "code": "",                    "name": ""                },                "primary": false            }        },        "routingNumber": "631916289",        "accountNumber": "708605532016",        "voucherID": "",        "address": {            "type": "",            "line1": "",            "line2": "",            "line3": "",            "line4": "",            "city": "",            "state": {                "code": "",                "name": ""            },            "province": {                "code": "",                "name": ""            },            "postalCode": "",            "country": {                "code": "",                "name": ""            },            "primary": false        }    },    "promocode": "H61MYC2HDS",    "email": "",    "phone": "",    "status": "canceled"}