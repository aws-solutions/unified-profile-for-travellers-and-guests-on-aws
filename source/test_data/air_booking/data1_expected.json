{"air_booking_recs": [{"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-JFK-HKG", "from": "JFK", "to": "HKG", "flight_number": "AA8944", "departure_date": "2023-04-02", "departure_time": "01:31", "arrival_date": "2023-04-02", "arrival_time": "06:28", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Joey", "middle_name": "Candelario", "last_name": "Wisozk", "gender": "female", "pronoun": "", "date_of_birth": "1934-05-19", "job_title": "Agent", "company": "Adobe Digital Government", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "melodyarmstrong@jenkins.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "87124 Rest berg", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Baton Rouge", "address_state_province": "", "address_postal_code": "62002", "address_country": "TO", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-HKG-MAD", "from": "HKG", "to": "MAD", "flight_number": "AA4343", "departure_date": "2023-04-02", "departure_time": "04:37", "arrival_date": "2023-04-02", "arrival_time": "10:00", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Joey", "middle_name": "Candelario", "last_name": "Wisozk", "gender": "female", "pronoun": "", "date_of_birth": "1934-05-19", "job_title": "Agent", "company": "Adobe Digital Government", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "melodyarmstrong@jenkins.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "87124 Rest berg", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Baton Rouge", "address_state_province": "", "address_postal_code": "62002", "address_country": "TO", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-MAD-FRA", "from": "MAD", "to": "FRA", "flight_number": "AA0447", "departure_date": "2023-04-02", "departure_time": "06:06", "arrival_date": "2023-04-02", "arrival_time": "16:40", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Joey", "middle_name": "Candelario", "last_name": "Wisozk", "gender": "female", "pronoun": "", "date_of_birth": "1934-05-19", "job_title": "Agent", "company": "Adobe Digital Government", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "melodyarmstrong@jenkins.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "87124 Rest berg", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Baton Rouge", "address_state_province": "", "address_postal_code": "62002", "address_country": "TO", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-JFK-HKG", "from": "JFK", "to": "HKG", "flight_number": "AA8944", "departure_date": "2023-04-02", "departure_time": "01:31", "arrival_date": "2023-04-02", "arrival_time": "06:28", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Joey", "middle_name": "Candelario", "last_name": "Wisozk", "gender": "female", "pronoun": "", "date_of_birth": "1934-05-19", "job_title": "Agent", "company": "Adobe Digital Government", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "melodyarmstrong@jenkins.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "87124 Rest berg", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Baton Rouge", "address_state_province": "", "address_postal_code": "62002", "address_country": "TO", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-HKG-MAD", "from": "HKG", "to": "MAD", "flight_number": "AA4343", "departure_date": "2023-04-02", "departure_time": "04:37", "arrival_date": "2023-04-02", "arrival_time": "10:00", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Joey", "middle_name": "Candelario", "last_name": "Wisozk", "gender": "female", "pronoun": "", "date_of_birth": "1934-05-19", "job_title": "Agent", "company": "Adobe Digital Government", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "melodyarmstrong@jenkins.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "87124 Rest berg", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Baton Rouge", "address_state_province": "", "address_postal_code": "62002", "address_country": "TO", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-MAD-FRA", "from": "MAD", "to": "FRA", "flight_number": "AA0447", "departure_date": "2023-04-02", "departure_time": "06:06", "arrival_date": "2023-04-02", "arrival_time": "16:40", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Joey", "middle_name": "Candelario", "last_name": "Wisozk", "gender": "female", "pronoun": "", "date_of_birth": "1934-05-19", "job_title": "Agent", "company": "Adobe Digital Government", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "melodyarmstrong@jenkins.net", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "87124 Rest berg", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Baton Rouge", "address_state_province": "", "address_postal_code": "62002", "address_country": "TO", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-JFK-HKG", "from": "JFK", "to": "HKG", "flight_number": "AA8944", "departure_date": "2023-04-02", "departure_time": "01:31", "arrival_date": "2023-04-02", "arrival_time": "06:28", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Ludwig", "middle_name": "Cordelia", "last_name": "Blanda", "gender": "female", "pronoun": "she", "date_of_birth": "1979-01-02", "job_title": "Designer", "company": "Barchart", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "", "email_type": "", "phone": "5327672053", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "personal", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "365 Lakes ton", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Birmingham", "address_state_province": "", "address_postal_code": "65579", "address_country": "SS", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-HKG-MAD", "from": "HKG", "to": "MAD", "flight_number": "AA4343", "departure_date": "2023-04-02", "departure_time": "04:37", "arrival_date": "2023-04-02", "arrival_time": "10:00", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Ludwig", "middle_name": "Cordelia", "last_name": "Blanda", "gender": "female", "pronoun": "she", "date_of_birth": "1979-01-02", "job_title": "Designer", "company": "Barchart", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "", "email_type": "", "phone": "5327672053", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "personal", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "365 Lakes ton", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Birmingham", "address_state_province": "", "address_postal_code": "65579", "address_country": "SS", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "booking_id": "TDRJ2F", "segment_id": "TDRJ2F-MAD-FRA", "from": "MAD", "to": "FRA", "flight_number": "AA0447", "departure_date": "2023-04-02", "departure_time": "06:06", "arrival_date": "2023-04-02", "arrival_time": "16:40", "channel": "ota-expedia", "status": "confirmed", "honorific": "Ms", "first_name": "Ludwig", "middle_name": "Cordelia", "last_name": "Blanda", "gender": "female", "pronoun": "she", "date_of_birth": "1979-01-02", "job_title": "Designer", "company": "Barchart", "price": 0.0, "nationality_code": "", "nationality_name": "", "language_code": "", "language_name": "", "pss_id": "", "gds_id": "", "payment_type": "cash", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_code": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "", "email_type": "", "phone": "5327672053", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "personal", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z", "address_line1": "365 Lakes ton", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Birmingham", "address_state_province": "", "address_postal_code": "65579", "address_country": "SS", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "address": "melodyarmstrong@jenkins.net", "type": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "address": "melodyarmstrong@jenkins.net", "type": "", "traveller_id": "4527724622", "last_updated_partition": "2022/06/22/23/12/30/3742435Z"}], "common_phone_recs": [{"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "number": "5327672053", "country_code": 32, "type": "personal", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "number": "858.432.5280", "country_code": 10, "type": "mobile", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z"}], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "id": "0518206991", "program_name": "ascend", "miles": 42587.0, "miles_to_next_level": 1667.0, "level": "silver", "joined": "2021-12-05T02:54:30.557043995Z", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-06-22T23:12:30.3742435Z", "last_updated_by": "Toni Tremblay", "id": "2447481825", "program_name": "elite", "miles": 945120.0, "miles_to_next_level": 6002.0, "level": "platinium", "joined": "2021-12-05T02:54:30.557043995Z", "traveller_id": "8924443001", "last_updated_partition": "2022/06/22/23/12/30/3742435Z"}]}